src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -106,6 +106,28 @@ public void sendUserRegistrationToChannel(UserDTO user, boolean trialAccountActi
         }
     }
 
+    @Async
+    public void sendUserApiAccessRequestToChannel(UserDTO user) {
+        log.debug("Sending notification to admin group that a user has requested API access");
+        if (StringUtils.isEmpty(this.applicationProperties.getSlack().getUserRegistrationWebhook())) {
+            log.debug("\tSkipped, the webhook is not configured");
+        } else {
+            List<LayoutBlock> layoutBlocks = new ArrayList<>();
+
+            String userPageLink = "(<https://www.oncokb.org/users/" + user.getEmail() + "/|" + user.getEmail() + ">)";
```

Comment:
We should update to use the base URL. The https://github.com/oncokb/oncokb-public/pull/1047 is merged.